platform/linux-generic/odp_thread.c

```diff
@@ -32,10 +33,10 @@ typedef struct {
 		odp_thrmask_t  control;
 	};
 
-	uint32_t       num;
-	uint32_t       num_worker;
-	uint32_t       num_control;
-	uint32_t       num_max;
+	odp_atomic_u32_t num;
+	odp_atomic_u32_t num_worker;
+	odp_atomic_u32_t num_control;
+	odp_atomic_u32_t num_max;
```

Comment:
Looks like `num_max` doesn't have to be atomic, since it's only modified in global init. Probably not much overhead but it's a bit confusing now when reading the code.

Reply:
Not atomic in v3.